templates/sign_in.html.hbs

```diff
@@ -1,12 +1,109 @@
-<!doctype html>
-<html>
-<body>
-<h1>Sign In</h1>
-<form method="POST" action="/auth/sign_in">
-    <input type="hidden" name="csrf_token" value="{{ token }}">
-    <input type="email" name="email" id="email" placeholder="E-mail address">
-    <input type="password" name="password" id="password" placeholder="Password">
-    <button>Log In</button>
-</form>
-</body>
-</html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Fedibook - Login</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.6.1/css/bulma.min.css">
+	<!-- This is a hacky CSS that really needs to be fixed/replaced. I'm just putting it here to see if I can improve the styling until proper Bulma implementation -->
+	<link rel="stylesheet" media="screen" href="../web_ui/stylesheets/scratchpad.css">
+  </head>
+  <body>
+  <!-- Does Bulma even understand "semantic web" stuff? Let's find out! -->
+  <!-- Begin Page Header -->
+	  <section class="hero is-dark">
+		  <!-- Hero head: will stick at the top -->
+		  <div class="hero-head">
+		  <h1 class="title">
+			Fedibook
+		  </h1>
+		  <h2 class="subtitle">
+			Codename: Gojira
+		  </h2>
+		</div>
+	  <div class="hero-foot is-right">
+		<div class="container">
+			<a href="sign_in.html" style="float: right">Have an Account? - Login</a>

```

Comment:
Seems to be backward...on `sign_in.html.hbs` there is a link to the "sign_in.html" page, and on the "sign_up.html.hbs" template there is a link to the "sign_up.html" page...should that be switched around?

Reply:
Fixed -.-